src/app/public/modules/select-field/select-field.component.ts

```diff
@@ -103,6 +103,13 @@ export class SkySelectFieldComponent implements ControlValueAccessor, OnDestroy
   @Input()
   public multipleSelectOpenButtonText: string;
 
+  /**
+   * When `inMemorySearchEnabled` is `false`, it will circumvent the list-builder search function,
+   * allowing consumers to provide results from a remote source, by updating the `data` value.
+   */
+  @Input()
+  public inMemorySearchEnabled: boolean;
```

Comment:
Is the default `true` then?

Reply:
Yep. I'll update the jsdoc comment to reflect that.